massifs/go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/datatrails/go-datatrails-merklelog/massifs
 
 go 1.22
 
+replace github.com/datatrails/go-datatrails-merklelog/mmr => ../mmr 
```

Comment:
minor: we should remove this replace

I did this accidentally before, the solution is once this PR is merged to release mmr and not massifs, then change massifs to depend on the new mmr version in a 1 line PR, once that is merged we can release massifs

Reply:
ah got it. so we merge. then we do tiny follow up release + re-merge. thats fine I can do that